astropy/io/misc/yaml.py

```diff
@@ -267,11 +266,11 @@ def _represent_tuple(self, data):
     AstropyDumper.add_representer(
         np_type, yaml.representer.SafeRepresenter.represent_int
     )
-for np_type in [np.float_, np.float16, np.float32, np.float64, np.longdouble]:
```

Comment:
So for users with numpy versions below 2 that used such data types, these will not be supported anymore?

Reply:
`np.float_` is just an alias of `np.float64`, i.e., if on numpy <2 you use it, it will still work:
```
In [7]: np.float_ is np.float64
Out[7]: True
```

Reply:
Ok, thanks for the clarification!